test/inject-encoding-test.js

```diff
@@ -64,7 +64,9 @@ buster.testCase('JSON API', {
 
   'simple-object keys in "json" encoding': function (done) {
     this.runTest([
-      { value: '0', key: 0 },
+      // Test keys that would be considered the same with default utf8 encoding.
+      // Because String([1]) === String(1).
+      { value: '0', key: [1] },
```

Comment:
This test was useless because it didn't actually test custom encoding of keys. The keyEncoding was in fact utf8.